src/compiler/parser.ts

```diff
@@ -9288,8 +9288,10 @@ namespace Parser {
 
             function parseJSDocLinkName() {
                 if (tokenIsIdentifierOrKeyword(token())) {
+                    if (token() === SyntaxKind.ImportKeyword) {
+                        return parseImportType();

```

Comment:
@sandersn should we reuse `parseImportType`, or would creating a new node to handle this case be better?